glide.yaml

```diff
@@ -6,10 +6,11 @@ import:
 - package: github.com/spf13/viper
 - package: github.com/pkg/errors
 - package: k8s.io/client-go
-  version: ^2.0.0
+  version: 3.0.0
   subpackages:
   - pkg/api/v1
-  - pkg/runtime
   - pkg/apis
+- package: k8s.io/apimachinery
+  version: release-1.6
```

Comment:
Why did we pin the version to "release-1.6"? client-go v3.0.0 does that already, right? Any reason we had to specify that explicitly here?